.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - node
```

Comment:
Any reason why we can't use 12? I'd like to stay on LTS since the end of life for 11 is quite soon (2019-06-01)

Reply:
@emilyemorehouse it can be reverted once the downstream issue is fixed. Another option would be to use a package-lock or npm-shrinkwrap to upgrade `brotli-size`

Reply:
Approve on using node 10.

There are several packaged still can not used with node 12. And we should keep the CI working at least. Otherwise contributors will be puzzled.

Reply:
Changed to 10 in 0430711, looks like CI is still working